README.md

```diff
@@ -1 +1,75 @@
-# -school-library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Decode morse code
+
+This project will make use of the Ruby language syntax to
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+## Built With
+
+- Ruby
+- Terminal
+- IRB
+- Rubocop (linters)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Install
+
+In your terminal, navigate to your current directory and run this code
+
+`https://github.com/betses/school-library.git`
+
+Locate the directory in your file explorer
+
+`cd school-library`
+
+Install Rubocop
+
+`bundle install`
+
+## Run Linters:
+
+### To run rubocop we use:
+
+`rubocop`
+
+### To autocorrect offenses with rubocop we use:
+
+`rubocop --auto-correct-all` or
+`rubocop -A`
+
+Have fun with morse code translation!
+
+## AuthorsBB;[]
+
+👤 **Betsegaw Sebsibe**
+
+- GitHub: [@BetsegawSebsibe ](https://github.com/betses)
+- Twitter: [@BetsegawSebsibe](https://twitter.com/Betse_s)
+- LinkedIn: [GordonOtieno](https://www.linkedin.com/in/betsegaw-sebsibe/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](/issues).
```

Comment:
Great job @betses 🤝

- [ ] [Optional] Kindly update the issues link to make your readme file more professional.